kalite/settings/base.py

```diff
@@ -314,8 +314,22 @@
 USE_L10N = getattr(local_settings, "USE_L10N", False)
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = getattr(local_settings, "SECRET_KEY",
-                     "8qq-!fa$92i=s1gjjitd&%s@4%ka9lj+=@n7a&fzjpwu%3kd#u")
+SECRET_KEY_FILE = os.path.join(USER_DATA_ROOT, "secretkey.txt")
```

Comment:
Yes, wondering what's going on here. Since `local_settings` is a dynamic object that may or may not be a module, the secret key is not all that reliable.

But is adding a file really that wise? What if it doesn't exist? I don't think that tests are gonna like this, nor do I think it's a viable pattern since settings should be load-able ALWAYS, especially since the `manage setup` command would depend on them.

Needs a fallback IMO.